disputer/disputer.js

```diff
@@ -237,12 +260,23 @@ class Disputer {
         continue;
       }
 
+      // Get resolved price request for dispute. `getPrice()` should not fail since the dispute price request must have settled in order for `withdrawLiquidation()`
+      // to be callable.
+      let resolvedPrice = await this.votingContract.getPrice(this.empIdentifier, requestTimestamp, {
```

Comment:
We do not need to catch this since `withdrawLiquidation` is only callable by the disputer IFF the dispute succeeded. Note, that this assumes that the disputer bot is NOT the liquidator. Is this a fair assumption?

Reply:
I'm a little confused on this one -- why does `getPrice` need to assume that?

Reply:
I'm not sure actually I forget why I wrote this comment haha